src/games/strategy/triplea/delegate/BattleDelegate.java

```diff
@@ -100,7 +100,8 @@ public void start() {
       addBombardmentSources();
       m_needToAddBombardmentSources = false;
     }
-    m_battleTracker.fightBombingRaids(m_bridge);
+    m_battleTracker.fightAirRaidsAndStrategicBombing(m_bridge);
```

Comment:
These two methods should be merged. We can have a bug if these two lines were re-ordered. Sensitivity to line ordering is a coding smell..